### ISSUE TITLE
Update script

### ISSUE THREAD

**Issue by ksilnoaa** [aomlomics/tourmaline#135]
Update [fastqc_per_base_sequence_quality_dropoff.py](https://github.com/aomlomics/tourmaline/blob/develop/scripts/fastqc_per_base_sequence_quality_dropoff.py) to work with fastq_summary.qzv output

**Comment by lukenoaa** [comment]
I agree it would be great if this script can work directly from QIIME output without running FastQC/MultiQC. The zip archive of fastq_summary.qzv does contain forward-seven-number-summaries.tsv and reverse-seven-number-summaries.tsv that contain the distribution of quality scores at each position. For example, we could have the script report at which position the 9th percentile drops below a quality score of 20.

**Comment by ksilnoaa** [comment]
@lukenoaa yes that's what I was thinking! Would you be able to adapt that script for that functionality? Then I can include it in the V2 workflow.
